enhancements/monitoring/user-workload-monitoring.md

```diff
@@ -173,27 +184,70 @@ Usage of the Thanos querier component inside cluster monitoring has many advanta
 
 See [5] and [6] for more details.
 
+#### Alerts/Recording Rules Aggregation
+
+Thanos Querier only solves aggregating the query path. For aggregating recording rules as well as alerting rules
+another component [Thanos Ruler](https://thanos.io/components/rule.md/) is deployed. This component is responsible for hosting user defined recording rules
+as well as alerting rules.
+
+Thanos Ruler is deployed via the user workload monitoring prometheus operator via a new CRD called ThanosRule.
+The same _deny list_ setting is applied to user workload monitoring prometheus operator such that the user cannot
+deploy its own Thanos Ruler instances in user namespaces.
+
+Thanos Ruler queries Thanos Querier to resolve recording and alerting rules.
+This does have implications on the availability of user defined alerts.
+In order for alerts to work the following systems need to be up:
```

Comment:
As discussed OOB: Lets just rephrase that these are the critical components but not all of them that need to be up an running for this to work.

Reply:
done